frontend/src/app/app.routes.ts

```diff
@@ -153,6 +154,10 @@ export const routes: Routes = [
             }
           }
         ]
+      },
+      {
+        path: 'cities',
+        component: CitiesUploadComponent
```

Comment:
Does it hurt? We hide the links that lead to these pages. So if a user (which has to be signed in), tries to access an admin page, he'll get to the page (unless there is a resolver which fails), and then won't be able to do anything. Not really an issue, is it?